lib/coophub/repos.ex

```diff
@@ -101,31 +105,25 @@ defmodule Coophub.Repos do
   @spec get_org_info(String.t()) :: org() | nil | :error
   def get_org_info(org_name) do
     case get_org(org_name) do
-      nil -> nil
-      :error -> :error
-      org -> org |> Map.delete("repos")
+      # Clear repos!
+      %Organization{} = org -> org |> Map.put(:repos, [])
```

Comment:
I don't `delete` because the property should be present at the struct (although it supports to be deleted).